### ISSUE TITLE
Not issue but a question

### ISSUE THREAD

**Issue by Tobe2d** [jmc2obj/j-mc-2-obj#259]
Is there a way to do it the other way around? from obj or blender into nbt?

**Comment by mmdanggg2** [comment]
No, not possible. Blender addon, perhaps, but that'd be a massive separate project.
